lms/djangoapps/certificates/models.py

```diff
@@ -253,6 +253,12 @@ def invalidate(self):
 
         self.save()
 
+    def is_valid(self):
+        """
+        Return True if certificate is valid and False if it is not valid.
```

Comment:
suggestion: just keep it `Return true if certificate is valid else return false.`